event/handler.go

```diff
@@ -0,0 +1,21 @@
+package event
+
+import (
+	"github.com/ONSdigital/go-ns/log"
+)
+
+// ObservationsImportedHandler ...
```

Comment:
Assuming this is only uncommennted/empty for now as this is the shell, the intention being this gets expanded in the implementation? I will inevitably be commenting that we don't need the constructor if everythings exported :)

Reply:
Yes the intention was to add comments alongside the implementation. The intention is to encapsulate the struct members which will require the constructor.